src/components/LabeledCheckbox/LabeledCheckbox.less

```diff
@@ -0,0 +1,41 @@
+.lucid-LabeledCheckbox {
+	align-items: center;
+	color: @color-darkGray;
+	cursor: pointer;
+	display: flex;
+	font: @font;
+	justify-content: flex-start;
+
+	&&-is-disabled {
+		color: @color-mediumGray;
+		cursor: not-allowed;
+	}
+
+	&:hover .lucid-Checkbox-visualization-container {
+		border-color: @color-primary;
+		transition: @Checkbox-animation-timing;
```

Comment:
Is there any special sort of scoping introduced by the compilers? Less by itself would put `@Checkbox-animation-timing` into the global scope in which case it would be usable regardless of order.